mart/models/modular.py

```diff
@@ -145,9 +145,13 @@ def training_step(self, batch, batch_idx):
                 )
             self.training_metrics(output[self.output_preds_key], output[self.output_target_key])
 
-        return output[self.output_loss_key]
+        loss = output[self.output_loss_key]
```

Comment:
What happens if loss is included in training_step_log?

Reply:
Then loss will be additionally logged as `training/loss` quietly (not on progress bar).